app/outputs/AWS-SES/Output.ts

```diff
@@ -15,6 +15,7 @@ export class AwsSesOutput extends OutputBase {
     constructor(formatter: FormatterBase, outputConfiguration: IAwsSesOutputConfiguration, appConfiguration: IAppConfiguration) {
         super(formatter, outputConfiguration, appConfiguration);
         this._configuration = outputConfiguration;
+    this.name = this.outputConfiguration.name;
```

Comment:
This line is incorrectly indented. 

```suggestion
        this.name = this.outputConfiguration.name;
```